index.js

```diff
@@ -0,0 +1,105 @@
+const URL = "https://trektravel.herokuapp.com/trips"
+//
+// Status Management
+//
+const reportStatus = (message) => {
+    $('#status-message').html(message);
+  };
+  
+  const reportError = (message, errors) => {
+    let content = `<p>${message}</p><ul>`;
+    for (const field in errors) {
+      for (const problem of errors[field]) {
+        content += `<li>${field}: ${problem}</li>`;
+      }
+    }
+    content += "</ul>";
+    reportStatus(content);
+  };
+
+
+const loadTrekTrips = () => {
+    reportStatus("Loading all trips...");
+    const tripsList = $('#trips-list')
+    tripsList.empty();
+    // tripsList.append('<th>All Trips</th>)')
+    axios.get(URL)
+    .then((response) => {
+      reportStatus(`Successfully loaded ${response.data.length} trips`);
+        console.log(response)
+        response.data.forEach((trip) => {
+            tripsList.append(`<li class='list-group-item text-center'>${trip.name}</li>`);
+        });
+    })
+    .catch((error) => {
+      reportStatus(`Encountered an error while loading wonders: ${error.message}`);
+      console.log(error);
+    });
+};
+
+const loadTripDetails = (index) => {
+    const tripDetails = $('#trip-details')
+    tripDetails.empty();
+    console.log(URL+"/"+(parseInt(index)+70).toString())
+    axios.get(URL+"/"+(parseInt(index)+70).toString())
+    .then((response) => {
+      console.log(response)
+      tripDetails.append(`<li><span>Name: </span> ${response.data.name}</li>`);
+      tripDetails.append(`<li><span>Continent: </span> ${response.data.continent}</li>`);
+      tripDetails.append(`<li><span>Category: </span> ${response.data.category}</li>`);
```

Comment:
this is enough to turn into a helper! It will be much easier to read afterwards!